app/2.0/start/toolbox/create-a-page.md

```diff
@@ -168,10 +169,11 @@ just need to add a menu item in the left-hand drawer so that users can navigate
     ...
     ```
 
-Your new page is now ready! Open your web browser and view it at
-[http://localhost:8080/new-view](http://localhost:8080/new-view).
+Your new page is now ready! Serve your app with `polymer serve`, then open the resulting URL in your browser.
+
+![Output from polymer serve command](/images/2.0/toolbox/polymer-serve-output.png)
```

Comment:
these images don't show up in preview. Could be a github issue, or the `app/` path prefix is missing